roles/dtc/common/templates/ndfc_vrf_lite_ebgp.j2

```diff
@@ -30,7 +30,7 @@ router bgp {{ MD_Extended.vxlan.global.bgp_asn }}
   {% endif %}
   {% if switch_item.bgp.best_path_as_path_relax %}
     bestpath as-path multipath-relax
-  {% elif switch_item.bgp.best_path_as_path_relax is not defined and (item.bgp.best_path_as_path_relax | default(defaults.vxlan.overlay_extensions.vrf_lites.bgp.graceful_restart)) %}
+  {% elif switch_item.bgp.best_path_as_path_relax is not defined and (item.bgp.best_path_as_path_relax | default(defaults.vxlan.overlay_extensions.vrf_lites.bgp.best_path_as_path_relax)) %}
```

Comment:
Was this originally a typo as the key is completely changing here?

Reply:
I assume it was originally a typo. I will let @devegupt confirm this.

Reply:
This was a typo, thanks for catching this.